src/transforms/corrcholeskylower.jl

```diff
@@ -0,0 +1,65 @@
+
+####
+#### correlation cholesky Lower
+####
+# Adapted from https://github.com/tpapp/TransformVariables.jl/blob/master/src/special_arrays.jl
+
+const TV = TransformVariables
+using LinearAlgebra
+
+"""
+    CorrCholeskyLower(n)
+Cholesky Lower of a correlation matrix of size `n`.
+Transforms ``n×(n-1)/2`` real numbers to an ``n×n`` lower-triangular matrix `L`, such that
+`L*L'` is a correlation matrix (positive definite, with unit diagonal).
+# Notes
+If
+- `z` is a vector of `n` IID standard normal variates,
+- `σ` is an `n`-element vector of standard deviations,
+- `L` is obtained from `CorrCholeskyLower(n)`,
+then `Diagonal(σ) * L * z` will be a multivariate normal with the given variances and
+correlation matrix `L * L'`.
+"""
+struct CorrCholeskyLower <: TV.VectorTransform
+    n::Int
+    function CorrCholeskyLower(n)
+        # @argcheck n ≥ 1 "Dimension should be positive."
+        new(n)
+    end
```

Comment:
This is just the default constructor
```suggestion
```

Reply:
Ah, right. I guess Tamas had this to have the `@argcheck` available. I'd love a nice way to have optional arg checks. There's
https://github.com/simeonschaub/OptionalArgChecks.jl
but it says it broken. Let's drop this for now and come back to the question of argchecks once we have more of the functionality in place